components/pages/workPageLayout/WorkPageHeader.tsx

```diff
@@ -1,3 +1,5 @@
+"use client"
```

Comment:
It was inherently a client component because it was imported into a client component. 

We use hooks in this component so it should always be a client component.